### ISSUE TITLE
JAWS 2020 not reading alertdialog content (Chrome, Edge, Firefox)

### ISSUE THREAD

**Issue by spmonahan** [FreedomScientific/VFO-standards-support#588]
### Summary

JAWS 2020 is only read the label of `alertdialog`s and not their content or descriptions.

Reproduction Steps
1. Go to JSFiddle example: https://jsfiddle.net/q7kvft51/show
2. Use JAWS to activate the `open alert dialog` button
3. JAWS reads the `aria-label` ("test") on the alertdialog but it does not read the dialog content ("some text").

### Expected result

JAWS should read both the label and the alertdialog content.

### Actual result

JAWS only reads the `aria-label` for the alertdialog.

### Example

https://jsfiddle.net/q7kvft51/show

#### JAWS version and build number

JAWS 2020.2110.36 ILM
 
#### Operating System and version

- OS: Windows 11 Enterprise
- Version: 10.0.22000 Build 22000

#### Browser and version:

- Chrome Version 95.0.4638.69 (Official Build) (64-bit)
- Edge Version 95.0.1020.44 (Official build) (64-bit)
- Firefox Version 94.0.1 (64-bit)

**Comment by JAWS-test** [comment]
Why would JAWS output the content? I don't think the ARIA specification requires this because alertdialog is not a live region:

> Authors SHOULD use aria-describedby on an alertdialog to reference the alert message element in the dialog. If they do not, an assistive technology can resort to its internal recovery mechanism to determine the contents of the alert message.

https://w3c.github.io/aria/#alertdialog

**Comment by smhigley** [comment]
JAWS has the same issue when `aria-describedby` is present: https://jsfiddle.net/3hu2z4c7/show

That said, `alertdialog` *is* a live notification region, as evidenced by [this text in the ARIA spec](https://w3c.github.io/aria/#alertdialog):
> The user agent SHOULD fire a system alert event through the accessibility API when the alert is created, provided one is specified by the intended accessibility API.

Or the "The user agent SHOULD fire a system alert event" mappings [in core AAM](https://w3c.github.io/core-aam/#mapping_role_table).

While both the linked fiddles work (in the sense that all text content is read) in JAWS 2020 and NVDA/Narrator (tested on Chrome and Edge), JAWS 2022 does not seem to be respecting either descriptions on a dialog, or the alert part of `alertdialog`. JAWS 2021 wasn't able to be tested by either @spmonahan or myself.

**Comment by JAWS-test** [comment]
I think that alertdialog is not a live region. In a live region, changed content would be output automatically. This is not what alertdialog is for. It is correct that an alert event should be triggered, but in my opinion this is not identical with a live region. For example, an alert event can cause JAWS to emit "Alert!" or a warning sound. This is not the case here. I.e. I would consider as an error:
* JAWS does not distinguish between dialog and alertdialog.
* JAWS does not trigger an alert event

It would be necessary to check if this is a JAWS or browser error (the alert event must come from the browser).

In the aria-describedby example, the description is output by JAWS 2018, 2019, 2020, but not by JAWS 2021.

The question is whether for dialog and alertdialog the description must be output when an element in dialog/alertdialog gets focus. It would be nice, but it is not required by the specification, I guess. The dialog/alertdialog label is output correctly when opened. For example, I can reference the dialog title and contents via aria-labelledby.
